### ISSUE TITLE
Cannot print inside Metal kernel, ignored

### ISSUE THREAD

**Issue by universewill** [taichi-dev/taichi#1281]
I want to print in ti.kernel, but get the message 'Cannot print inside Metal kernel, ignored'.

Here is the code:
```
import taichi as ti

print(ti.__version__)
ti.init(ti.gpu)
pixels = ti.var(dt=ti.f32, shape=(640, 320))

@ti.kernel
def paint(t:ti.f32):
    print(t)
paint(0.3)
```

**Comment by archibate** [comment]
Metal backend doesn't support `print` for now, use `ti.init(ti.cpu)` if you need to.
@k-ye Could you add print support to Metal now? Given the success of adding print to OpenGL, it's won't be hard, right?

**Comment by archibate** [comment]
Resolved in #1310.
